content/join-coreteam.md

```diff
@@ -19,4 +19,6 @@ Coreteam is organised as a flat structure under a single president (instead of i
 
 ## Want to join us?
 
-Just [email]({{< ref "/contact" >}}) or approach any one of us&mdash;we'll set up an interview to find out more about what you're interested in doing. This is usually an informal chat over lunch, so don't worry too much about it; it does help to take part in our events and check out the rest of this site beforehand, though. This is also an opportunity for you to get to know us better, so feel free to ask any questions, too.
+You can sign up using this [form](https://forms.gle/GfYbNgFYdEeBHA8DA).
```

Comment:
Can you add the PDPA thing to the form?

Reply:
ohno

Reply:
ok done boss @chrisgzf can you take a look

Reply:
okie i added a separate qn to ensure that people acknowledge should be okay alr